xbmc/linux/XFileUtils.cpp

```diff
@@ -239,7 +239,7 @@ DWORD  SetFilePointer(HANDLE hFile, int32_t lDistanceToMove,
 }
 
 // uses statfs
-BOOL GetDiskFreeSpaceEx(
+bool GetDiskFreeSpaceEx(
```

Comment:
This breaks the "emulation" as the method signature isn't the same, however this method is unused I think